components/paging/pager-load-more.js

```diff
@@ -129,20 +130,33 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
 
 		const item = pageable._getItemByIndex(lastItemIndex + 1);
 
-		if (!item) return;
-		if (item.updateComplete) await item.updateComplete;
-
-		if (item.focus !== nativeFocus) {
-			requestAnimationFrame(() => item.focus());
-		} else {
-			const firstFocusable = getFirstFocusableDescendant(item);
-			if (firstFocusable) {
-				firstFocusable.focus();
-			} else if (item.focus === nativeFocus) {
-				item.tabIndex = -1;
-				requestAnimationFrame(() => item.focus());
+		let itemToFocus;
+		if (item) {
+			if (item.updateComplete) await item.updateComplete;
+			if (item.focus !== nativeFocus) {
+				itemToFocus = item;
+			} else {
+				const firstFocusable = getFirstFocusableDescendant(item);
+				if (firstFocusable) {
+					itemToFocus = firstFocusable;
+				} else if (item.focus === nativeFocus) {
+					item.tabIndex = -1;
+					itemToFocus = item;
+				}
 			}
 		}
+
+		if (itemToFocus) {
+			await new Promise(requestAnimationFrame);
+			itemToFocus.focus();
+		}
+
+		await new Promise(setTimeout);
```

Comment:
This timeout is the ensure that focus has moved... should it be an animation frame instead?

Reply:
I think so yeah.

Btw, I noticed a line further above that could probably be simplified.

```javascript
await new Promise(resolve => requestAnimationFrame(resolve));
```
Pretty sure I wrote that... not sure why I didn't write it the way you did on line 150. I can't imagine the timing being different.

Reply:
Cool -- fixed.